web/src/filter.js

```diff
@@ -593,6 +597,7 @@ export class Filter {
         // stream, stream + topic,
         // is:dm, dm,
         // is:resolved
+        // sender
```

Comment:
> Let's work to update this comment in the prep commit so that we don't list out all the filters here.

I am not sure what you mean over here, could you please elaborate.

> Also, the comment above is_common_narrow with the link to the paper document will become even more out of date with these changes, so we should figure out how we want to deal with that. We could either update that document or update the comment here to better describe what a common narrow is. Maybe worth bringing up in the CZO conversation about this refactor.

Started the discussion: [CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/is_common_narrow.20rewrite/near/1627630)